Userland/Libraries/LibGL/Buffer/Buffer.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2022, cflip <cflip@cflip.net>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "Buffer.h"
+#include <AK/Format.h>
+
+namespace GL {
+
+void Buffer::set_data(void const* data, size_t size)
+{
+    if (!data) {
+        m_data = MUST(ByteBuffer::create_uninitialized(size));
+        return;
+    }
+    m_data = MUST(ByteBuffer::copy(data, size));
```

Comment:
Can we make this method fallible (i.e. `TRY`) so we can start using `GL_OUT_OF_MEMORY` in the future?